portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,68 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
+    <div id="intro-content">
       <img src="/images/PROFILE_PIC.JPG">
       <h1>Michaela Murray's Portfolio</h1>
-      <p id="intro"> Welcome! My name is Michaela. I am currently a rising junior at Stanford University!</p>
-      <p>Click here to get a random message:</p>
+      <p id="intro">
+        <br>Rising Junior @ Stanford University</br> 
+        <br>Major: Computer Science</br>
+        <br>STEP Intern @ Google</br>
```

Comment:
I think `br` is an empty element tag. Would this be better modeled as a list?

Reply:
Yes it would! Updated!

Reply:
Cool! Mind fixing the indentation for the list?

Reply:
Just fixed indentation in the index.html!